charts/dapr/charts/dapr_operator/templates/dapr_operator_service.yaml

```
@@ -7,9 +7,10 @@ metadata:
     {{- range $key, $value := .Values.global.k8sLabels }}
     {{ $key }}: {{ tpl $value $ }}
     {{- end }}
-{{- if .Values.apiService.annotations }}
-{{ toYaml .Values.apiService.annotations | indent 4}}
-{{- end }}
+  {{- if .Values.apiService.annotations }}
+  annotations:
+  {{ toYaml .Values.apiService.annotations | indent 4}}
```

You must add `-` at the start or the first line will have indentation of 6 (and the second of 4)
```suggestion
  {{- toYaml .Values.apiService.annotations | indent 4}}
```